R/release.R

```diff
@@ -104,9 +97,7 @@ release_checklist <- function(version, on_cran) {
     "Prepare for release:",
     "",
     todo("`git pull`"),
-    if (!is.na(milestone_num)) {
-      todo("[Close v{version} milestone](../milestone/{milestone_num})")
-    },
+    todo("[Close v{version} milestone](../milestone/{milestone_num})", !is.na(milestone_num)),
```

Comment:
I think this was just a case of forgetting that `todo()` can be called this way.